docs/sources/dashboards/build-dashboards/_index.md

```diff
@@ -24,4 +24,9 @@ This section includes the following topics:
 
 ## Dynamic dashboards
 
-You can create more interactive and dynamic dashboards by adding and using [variables]({{< relref "../variables" >}}). Instead of hard-coding things like server, application, and sensor names in your metric queries, you can use variables in their place. Read more about variables [here]({{< relref "../variables" >}}).
+You can create more interactive and dynamic dashboards by adding and using [variables][]. Instead of hard-coding things like server, application, and sensor names in your metric queries, you can use variables in their place. Read more about variables [here][].
```

Comment:
This link doesn't work. Rendering as just: [here][]
```suggestion
You can create more interactive and dynamic dashboards by adding and using [variables][]. Instead of hard-coding things like server, application, and sensor names in your metric queries, you can use variables in their place. Read more about variables [here][].
```